docs/installation_configuration_guide/Common.md

```diff
@@ -4,6 +4,8 @@ sidebar_position: 1
 ---
 # Configuration
 
+## Minimal configuration file
```

Comment:
I need a link to here, we use this term further on and it is an important concept.